### ISSUE TITLE
Connect Google Colab JSONDecodeError: Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by kienvht** [ultralytics/hub#38]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar bug report.

### HUB Component

Models

### Bug

First Step: I have create colab and run:
```
%pip install ultralytics  # install
import ultralytics
ultralytics.checks()  # checks
```
-> Success

Second step: 
```
ultralytics.start('XXXXXX')  # my API key
```

-> Get error
```
[/usr/local/lib/python3.7/dist-packages/ultralytics/__init__.py](https://localhost:8080/#) in start(key)
     59     # key = 'ac0ab020186aeb50cc4c2a5272de17f58bbd2c0.RqFCDNBxgU4mOLmaBrcd'  # example
     60     # api_key='ac0ab020186aeb50cc4c2a5272de17f58bbd2c0', model_key='RqFCDNBxgU4mOLmaBrcd'  # example
---> 61     train_model(api_key=api_key, model_key=model_key)

[/usr/local/lib/python3.7/dist-packages/ultralytics/main.py](https://localhost:8080/#) in train_model(api_key, model_key)
     15     # connect_to_hub(api_key=api_key)  # deprecated
     16     AUTH.api_key = api_key
---> 17     trainer = Trainer(model_id=model_key, auth=AUTH)  # No model so next in train queue is fetched
     18     if trainer.model is not None:
     19         trainer.start()

[/usr/local/lib/python3.7/dist-packages/ultralytics/trainer.py](https://localhost:8080/#) in __init__(self, model_id, auth)
     12     def __init__(self, model_id, auth):
     13         self.auth = auth
---> 14         self.model = self._get_model(model_id)
     15         if self.model is not None:
     16             self._connect_callbacks()

[/usr/local/lib/python3.7/dist-packages/ultralytics/trainer.py](https://localhost:8080/#) in _get_model(self, model_id)
     33         try:
     34             r = requests.post(api_url, json=payload)
---> 35             res = r.json()
     36 
     37             if res["data"] is None:

[/usr/local/lib/python3.7/dist-packages/requests/models.py](https://localhost:8080/#) in json(self, **kwargs)
    896                     # used.
    897                     pass
--> 898         return complexjson.loads(self.text, **kwargs)
    899 
    900     @property

[/usr/lib/python3.7/json/__init__.py](https://localhost:8080/#) in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    346             parse_int is None and parse_float is None and
    347             parse_constant is None and object_pairs_hook is None and not kw):
--> 348         return _default_decoder.decode(s)
    349     if cls is None:
    350         cls = JSONDecoder

[/usr/lib/python3.7/json/decoder.py](https://localhost:8080/#) in decode(self, s, _w)
    335 
    336         """
--> 337         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    338         end = _w(s, end).end()
    339         if end != len(s):

[/usr/lib/python3.7/json/decoder.py](https://localhost:8080/#) in raw_decode(self, s, idx)
    353             obj, end = self.scan_once(s, idx)
    354         except StopIteration as err:
--> 355             raise JSONDecodeError("Expecting value", s, err.value) from None
    356         return obj, end

JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

### Environment

```
Ultralytics HUB Version
v0.1.14-beta.1
Client User Agent
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.67 Safari/537.36
Operating System
Win32
Browser Window Size
1920 x 969
Server Timestamp
1653926780
```

**Comment by glenn-jocher** [comment]
@kienvht I took a look and was able to train this model successfully. Can you please retry and let us know if you continue having problems?

**Comment by kienvht** [comment]
Run successfully with `ultralytics 0.0.24`

**Comment by UltralyticsAssistant** [comment]
@kienvht great to hear that the issue has been resolved with `ultralytics 0.0.24`. If you encounter any further problems or if you have additional questions, feel free to reach out. The YOLO community and the Ultralytics team are always here to help. For more detailed information and documentation, please visit the Ultralytics HUB Docs. Happy coding!
